File: backend/src/test/java/com/skewwhiffy/auraltester/notation/model/note/NoteTest.java
/*
package com.skewwhiffy.notation.model.note

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class NoteTest {
  @Test
  fun `return display string when natural`() {
    val note = Note.a

    val actual = note.displayString

    assertThat(actual).isEqualTo("A")
  }

  @Test
  fun `return display string when sharp`() {
    val note = Note.b.sharp

    val actual = note.displayString

    assertThat(actual).isEqualTo("B#")
  }

  @Test
  fun `return display string when flat`() {
    val note = Note.d.flat

    val actual = note.displayString

    assertThat(actual).isEqualTo("Db")
  }

  @Test
  fun `equate equivalent notes`() {
    fun getNote() = Note("A", Accidental.sharp)

    val first = getNote()
    val second = getNote()

    assertThat(first).isEqualTo(second)
    assertThat(first <= second).isTrue
    assertThat(first >= second).isTrue
  }

  @Test
  fun `compare non-equivalent notes`() {
    val lower = Note.c

    val higher = Note.a

    assertThat(lower < higher).isTrue
    assertThat(lower > higher).isFalse
    // TODO: > and < with #s and bs
  }
}*/

File: backend/src/test/kotlin/com/skewwhiffy/auraltester/notation/model/note/NoteTest.kt
package com.skewwhiffy.auraltester.notation.model.note
/*

import lombok.`val`
internal class NoteTest {
    @org.junit.jupiter.api.Test
    fun returnsDisplayStringWhenNatural() {
        val note: `val` = Note.getA()
        val actual: `val` = note.getDisplayString()
        org.assertj.core.api.Assertions.assertThat<`val`>(actual).isEqualTo("A")
    }

    @org.junit.jupiter.api.Test
    fun returnsDisplayStringWhenSharp() {
        val note: `val` = Note.getB().sharpen()
        val actual: `val` = note.getDisplayString()
        org.assertj.core.api.Assertions.assertThat<`val`>(actual).isEqualTo("B#")
    }

    @org.junit.jupiter.api.Test
    fun returnsDisplayStringWhenFlat() {
        val note: `val` = Note.getD().flatten()
        val actual: `val` = note.getDisplayString()
        org.assertj.core.api.Assertions.assertThat<`val`>(actual).isEqualTo("Db")
    }

    @org.junit.jupiter.api.Test
    fun equatesEquivalentNotes() {
        val getNote = java.util.function.Supplier<Note> { Note("A", Accidental.getSharp()) }
        val first: `val` = getNote.get()
        val second: `val` = getNote.get()
        org.assertj.core.api.Assertions.assertThat<`val`>(first).isEqualTo(second)
        org.assertj.core.api.Assertions.assertThat<`val`>(first).isGreaterThanOrEqualTo(second)
        org.assertj.core.api.Assertions.assertThat<`val`>(first).isLessThanOrEqualTo(second)
    }

    @org.junit.jupiter.api.Test
    fun compareNonEquivalentNotes() {
        val lower: `val` = Note.getC()
        val higher: `val` = Note.getA()
        org.assertj.core.api.Assertions.assertThat<`val`>(lower).isLessThan(higher)
    } // TODO: > and < with #s and bs
}


 */